packages/@aws-cdk-testing/framework-integ/test/aws-stepfunctions-tasks/test/glue/integ.glue-task.js.snapshot/aws-stepfunctions-integ.template.json

```diff
@@ -1,253 +1,253 @@
 {
- "Resources": {
-  "GlueJobRole1CD031E0": {
-   "Type": "AWS::IAM::Role",
-   "Properties": {
-    "AssumeRolePolicyDocument": {
-     "Statement": [
-      {
-       "Action": "sts:AssumeRole",
-       "Effect": "Allow",
-       "Principal": {
-        "Service": "glue"
-       }
+  "Resources": {
+    "GlueJobRole1CD031E0": {
+      "Type": "AWS::IAM::Role",
+      "Properties": {
+        "AssumeRolePolicyDocument": {
+          "Statement": [
+            {
+              "Action": "sts:AssumeRole",
+              "Effect": "Allow",
+              "Principal": {
+                "Service": "glue.amazonaws.com"
```

Comment:
Were these glue service principals simply wrong before?

Reply:
Correct, this policy wasn't actually in use in the test so it wasn't caught but it was invalid altogether.